CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-# [# 0.8.0 (2020-05-23) - BigInt, Modularized Parser, Faster Hashing](https://github.com/jasonwilliams/boa/compare/v0.7.0...HEAD)
+# [# 0.8.0 (2020-05-23) - BigInt, Modularized Parser, Faster Hashing](https://github.com/boa-dev/boa/compare/v0.7.0...HEAD)
```

Comment:
This should probably be changed for the diff between 0.7 and 0.8, not the head.

Reply:
> 
> 
> Looks good! Do we have the correct link for the repo in boa/Cargo.toml?

My bad. Now it does.

> 
> 
> This should probably be changed for the diff between 0.7 and 0.8, not the head.

Fixed!